src/app/options.cpp

```diff
@@ -235,6 +240,69 @@ namespace
         return o == s;
     }
 
+    // Option that is explicitly set to true or false, and whose value is undefined when unspecified.
+    // May not have a shortcut.
+    class TriStateBoolOption: protected StringOption
+    {
+    public:
+        constexpr TriStateBoolOption(const char *name)
+            : StringOption {name}
+        {
+        }
+
+        using StringOption::operator==;
+        
+        QString usage() const
+        {
+            return StringOption::usage("true|false");
+        }
+
+        TriStateBool value(const QString &arg) const
+        {
+            QString val = StringOption::value(arg);
+            
+            if (val.toUpper() == "TRUE" || val == "1") {
+                return TriStateBool::True;
+            }
+            else if (val.toUpper() == "FALSE" || val == "0") {
+                return TriStateBool::False;
+            }
+            else {
+                //: Translation must use English "true" and "false"
```

Comment:
I haven't used Transifex before so I'm not sure. But this is the officially supported way of adding comments to translatable strings according to Qt's documentation. Is there another method I should use?